src/WebJobs.Script.WebHost/Diagnostics/Extensions/ScriptHostServiceLoggerExtension.cs

```diff
@@ -332,7 +332,13 @@ public static void ScriptHostServiceRestartCanceledByRuntime(this ILogger logger
 
         public static void LogHostInitializationSettings(this ILogger logger, string functionWorkerRuntime, string functionExtensionVersion, string currentDirectory, bool inStandbyMode)
         {
-            _logHostInitializationSettings(logger, functionWorkerRuntime, functionExtensionVersion, currentDirectory, inStandbyMode, null);
+            var hostInitializationSettings = $@"Settings: {{
```

Comment:
Instead of "Settings", should we use a more specific term here?

Reply:
Updated!